packages/venia-ui/lib/components/AccountMenu/__tests__/__snapshots__/accountMenuItems.spec.js.snap

```diff
@@ -2,13 +2,13 @@
 
 exports[`it renders correctly 1`] = `
 <div>
-  &lt;Link&gt;Order History&lt;/Link&gt;
```

Comment:
Is this change intentional? Should we not still have a string represented in the snapshot?